### ISSUE TITLE
Enforce all "harness" commands to use LANG=C.UTF-8

### ISSUE THREAD

**Issue by happz** [teemtee/tmt#3002]
All commands invoked by tmt for its own needs shall enforce locale to get reliable, parseable output. Do not apply to test commands and scripts though.

**Comment by psss** [comment]
Yeah, this definitely makes sense and is needed to get predictable outputs. Adding to the backlog.
